runners/direct-java/build.gradle

```diff
@@ -43,16 +51,40 @@ dependencies {
   shadow library.java.slf4j_api
   shadow library.java.hamcrest_core
   shadow library.java.junit
-  testCompile project(path: ":sdks:java:core", configuration: "shadowTest")
-  testCompile project(":runners:core-java").sourceSets.test.output
-  testCompile project(":model:fn-execution").sourceSets.test.output
-  testCompile library.java.guava_testlib
-  testCompile library.java.slf4j_jdk14
-  testCompile library.java.mockito_core
-  testCompile library.java.stax2_api
-  testCompile library.java.woodstox_core_asl
-  testCompile library.java.google_cloud_dataflow_java_proto_library_all
-  testCompile library.java.jackson_dataformat_yaml
+  shadowTest project(path: ":sdks:java:core", configuration: "shadowTest")
+  shadowTest project(":runners:core-java").sourceSets.test.output
+  shadowTest project(":model:fn-execution").sourceSets.test.output
+  shadowTest library.java.guava_testlib
+  shadowTest library.java.slf4j_jdk14
+  shadowTest library.java.mockito_core
+  shadowTest library.java.stax2_api
+  shadowTest library.java.woodstox_core_asl
+  shadowTest library.java.google_cloud_dataflow_java_proto_library_all
+  shadowTest library.java.jackson_dataformat_yaml
+  needsRunner project(path: ":runners:core-construction-java", configuration: "shadowTest")
+  needsRunner project(path: ":runners:core-java", configuration: "shadowTest")
+  needsRunner project(path: ":sdks:java:core", configuration: "shadowTest")
+  needsRunner project(path: project.path, configuration: "shadow")
+  needsRunner project(path: project.path, configuration: "shadowTest")
+}
+
+task needsRunnerTests(type: Test) {
+  group = "Verification"
+  description = "Runs tests that require a runner"
```

Comment:
update description to "Run tests that require a runner to validate pipelines/transforms work correctly."

Reply:
Done. I left it as "runs" rather than "run" to match the other task descriptions.